docs/en/portal/apollo-open-api-platform.md

```
@@ -725,6 +734,125 @@ App can be created through this interface,
+* **Request Params** :
+
+| Parameter Name       | Required | Type    | Description                                      |
+| -------------------- | -------- | ------- | ------------------------------------------------ |
+| keyword              | true     | String  | Match against username, display name, or email    |
+| includeInactiveUsers | false    | Boolean | Include disabled users; default `false`          |
+| offset               | false    | Integer | Offset; default `0`                              |
+| limit                | false    | Integer | Page size; default `10`                          |
+
+* **Request Sample** :
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)